cluster-provision/k8s/1.33/k8s_provision.sh

```diff
@@ -13,7 +13,7 @@ function getKubernetesClosestStableVersion() {
     kubernetes_minor_version=$(echo $kubernetes_version | cut -d. -f2)
     packages_major_version=$(echo $kubernetes_version | cut -d. -f1)
     packages_minor_version=$((kubernetes_minor_version-1))
-    packages_version="$(curl --fail -L "https://storage.googleapis.com/kubernetes-release/release/stable-${packages_major_version}.${packages_minor_version}.txt" | sed 's/^v//')"
+    packages_version="$(curl --fail -L "https://raw.githubusercontent.com/kubevirt/kubevirtci/refs/heads/main/cluster-provision/k8s/1.${packages_minor_version}/version")"
```

Comment:
I am not sure whether we should use ourselves as a source of truth around k8s release versions - is there no other way to determine the latest stable k8s version from any kubernetes release resource?

Reply:
I thought this would be a good solution as that is the latest known good version that we are testing with for the previous provider. 

I wouldn't consider it a source of truth - in this case we just need some recent version to install the rpms - the binaries get replaced later anyway.

Reply:
I think I have found a better option.